src/store/modules/site_config.js

```diff
@@ -15,6 +15,10 @@ const state = {
     did_config_load_yet: false,
 
     selected_site: '',
+    selected_subpage: 'home',
+    selected_target_tab: 'telescope controls', 
+    selected_image_tools_tab: 'analysis',
+    selected_controls_tab: 5, 
```

Comment:
Since these aren't very related to the site config, I think it would be better to put them in a different module. Perfectly fine if you want to make a new one to keep track of UI state like this. I see why it might seem like site_config is the place to put them, since other user selections (selected_enclosure, etc) are here, but those are used specifically for parsing the config files 🙃